client/source/components/RR/styles/RR.styled.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components';
+
+export const ReviewModuleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  border: 0.1em solid black;
+`;
+
+export const ReviewListContainer = styled.div`
+
+`;
+
+// export const ReviewTileContainer = styled.div`
```

Comment:
No need to leave development-only comments in a merge to the production branch.

Reply:
Updated